src/main/scala/com/typesafe/sbt/packager/archetypes/JavaServerApplication.scala

```diff
@@ -59,27 +57,26 @@ object JavaServerAppPackaging extends AutoPlugin {
     },
     // === etc config mapping ===
     bashScriptEnvConfigLocation := Some("/etc/default/" + (packageName in Linux).value),
-    linuxEtcDefaultTemplate <<= sourceDirectory map { dir =>
```

Comment:
No, they didn't work anymore in the global config scope since I added the dependency for `serverLoading`, and that config key is not defined globally.